src/main/java/com/purbon/kafka/topology/model/Topic.java

```diff
@@ -50,4 +63,6 @@
   void setMetadata(Map<String, String> metadata);
 
   SubjectNameStrategy getSubjectNameStrategy();
+
+  void setSubjectNameStrategy(Optional<SubjectNameStrategy> subjectNameStrategy);
```

Comment:
why is the setter need?

Reply:
When changing `TopicDeserializer` from `TopicImpl` to `Topic`, these method was only available in the implementation.